lib/styles/settings/_buttons.scss

```diff
@@ -48,3 +49,162 @@ $regular-colors: (
 	'warning': $color-warning,
 	'success': $color-success,
 );
+
+$only-icon-button-colors: (
+	'neutral-weak': (
+		'color': $color-default-text-inverted,
+		'icon': $color-default-icon-inverted,
+		'background': $color-neutral-background-strong,
+		'hovered': $color-neutral-background-strong-hovered,
+		'focused': $color-neutral-background-strong-focused,
+		'disabled': $color-neutral-background-strong-disabled,
+		'ripple': $color-default-ripple-inverted,
+		'inverted': (
```

Comment:
UX uses `inverted` work with different meaning. IMO, we should think of a better name here